packages/ui/src/Switch/Switch.tsx

```diff
@@ -1,36 +1,63 @@
 import React from "react";
-import { Switch as RmwcSwitch, SwitchProps } from "@rmwc/switch";
+import { Switch as SwitchBase } from "@webiny/admin-ui";
 import { FormComponentProps } from "~/types";
-import pick from "lodash/pick";
 import { FormElementMessage } from "~/FormElementMessage";
-import { getClasses } from "~/Helpers";
 
-type Props = Omit<SwitchProps, "value"> &
-    FormComponentProps<boolean> & {
-        // Description beneath the switch.
-        description?: string;
+type Props = {
+    /** Unique identifier for the control. */
+    id?: string;
 
-        // Optional class name.
-        className?: string;
-    };
+    /** Disables the control when set to true. */
+    disabled?: boolean;
+
+    /** Sets the control to checked (on) or unchecked (off). */
+    checked?: boolean;
+
+    /** The value of the control. Can be a string, number, or an array of strings. */
+    value?: string | number | string[];
+
+    /** A label displayed alongside the control. Can be any React node. */
+    label?: React.ReactNode;
+
+    /** Additional description displayed beneath the control. */
+    description?: string;
+
+    /** Additional CSS classes to apply to the control. */
+    className?: string;
+} & FormComponentProps<boolean>;
 
 /**
- * Switch component can be used to store simple boolean values.
+ * @deprecated This component is deprecated and will be removed in future releases.
+ * Please use the `<Switch />` component from the `@webiny/admin-ui` package instead.
```

Comment:
BTW something super minor I noticed before, but forgot to mention. 

```suggestion
 * Please use the `Switch` component from the `@webiny/admin-ui` package instead.
```

Basically <Switch /> is actual component written in JSX.

Reply:
Ok, I will apply these changes to other components too.